security/xca/PortFile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                  1.0
+
+name                        xca
+version                     2.1.2
+
+categories                  security crypto
+platforms                   darwin
+license                     xca
+
+maintainers                 {@lhaeger} openmaintainer
+
+description                 X Certificate and Key management
+long_description            XCA is a CA & PKI management tool that supports creating and managing X.509 certificates, \
+                            certificate requests, RSA, DSA and EC private keys, Smartcards and CRLs.
+homepage                    https://www.hohnstaedt.de/xca/
+
+PortGroup                   github 1.0
+
+github.setup                chris2511 ${name} ${version} RELEASE.
+github.tarball_from         releases
+
+checksums                   rmd160  16717ab6eb2cd33599d9606b7908551ad01bbff3 \
+                            sha256  fc845470a02b0b4534b46590be307f784662071fc412fdcad605c3bce901fe05 \
+                            size    1086563
+
+PortGroup                   qt5 1.0
```

Comment:
PortGroups are normally placed at the top of the Portfile. Move this line just below the GitHub Portgroup line.